data/json/overmap/overmap_terrain/overmap_terrain.json

```diff
@@ -416,9 +416,9 @@
   {
     "type": "overmap_terrain",
     "id": [ "bandit_garage_1", "bandit_garage_2" ],
-    "name": "forest",
-    "sym": "F",
-    "color": "green",
+    "name": "garage",
+    "sym": "O",
+    "color": "white",
     "see_cost": 5,
     "flags": [ "RISK_LOW", "SOURCE_SAFETY", "SOURCE_FOOD" ]
```

Comment:
SOURCE_FOOD, when more obviously a garage?

Reply:
As far I understand it, those flags are there for the NPCs, and since the bandits are NPCs they would disperse if they thought somewhere elsr was more shiny. Didn't test it in any capacity, but they seemed to sit on their butt and await company so I didn't touch a thing.